privaterelay/apps.py

```diff
@@ -10,10 +13,80 @@
 ROOT_DIR = os.path.abspath(os.curdir)
 
 
+class VersionInfo(TypedDict):
+    source: str
+    version: str
+    commit: str
+    build: str
+
+
+@cache
+def get_version_info(base_dir: str | Path | None = None) -> VersionInfo:
+    """Return version information written by build process."""
+    if base_dir is None:
+        base_path = Path(settings.BASE_DIR)
+    else:
+        base_path = Path(base_dir)
+    version_json_path = base_path / "version.json"
+    info = {}
+    if version_json_path.exists():
+        with version_json_path.open() as version_file:
+            try:
+                info = json.load(version_file)
+            except ValueError:
+                pass
+            if not hasattr(info, "get"):
+                info = {}
+    version_info = VersionInfo(
+        source=info.get("source", "https://github.com/mozilla/fx-private-relay"),
+        version=info.get("version", "unknown"),
+        commit=info.get("commit", "unknown"),
+        build=info.get("build", "not built"),
+    )
+    return version_info
+
+
+def get_profiler_startup_data() -> Tuple[str | None, str | None, str | None]:
+    if settings.RELAY_CHANNEL not in ("dev", "stage", "prod"):
+        return (None, None, None)
+
+    if settings.RELAY_CHANNEL in ("dev", "stage"):
+        service = "fxprivaterelay-nonprod"
```

Comment:
Q: Can we derive `service` from the `RELAY_CHANNEL`? Like `f"fxprivaterelay-{settings.RELAY_CHANNEL}"`. It looks like we get to choose this name from the docs [Service name and version arguments](https://cloud.google.com/profiler/docs/profiling-python).